admin_manual/images/appliance/setup/readme.md

```diff
@@ -0,0 +1 @@
+This are images of the setup
```

Comment:
s/This/These/g

Reply:
The text inside this readme is completely irrelevant :D
I just had to create a file for the folder creation.